.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!-- Just like NASA going to the moon, it's always good to have a checklist when
+creating changes.
+The following items are listed to help you create a great Pull Request: -->
+## Checklist
+- [ ] All new and existing **tests are passing**
+  - [ ] Python tests using `make test flakes` are passing
+  - [ ] Javascript unit tests using `npm run test-unit` are passing
+  - [ ] Javascript integration tests using `npm run test-integration` are passing
+- [ ] I have added tests to cover my changes
+- [ ] My change requires a change to the **documentation**.
+  - [ ] I have updated the documentation accordingly.
+  - [ ] I have added an entry to `CHANGES` because this is a user-facing change or an important bugfix
+- [ ] I have written **proper commit message(s)**
+      <!-- Title under 50 characters, good explanations, affected part of Isso
```

Comment:
Should 50 be the limit? Maybe 72 would be better since sometimes it's hard to fit everything under 50 characters. Many commits already on the repo are already about 55-60 characters long.

Maybe this could be reworded to "ideally under 50 characters, but no more than 72"?

Reply:
Thanks, fixed.